### PR TITLE
Add bitmanip doc

### PR DESCRIPTION
**Pull request by weicaiyang**
For task #520

### CODE REVIEW

**Comment by taoliug** [comment]
Can you the makefile under doc directory to regenerate the singlehtml doc?

File: docs/source/extension_support.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+.. _riscv-ovpsim: https://github.com/riscv/riscv-ovpsim
+
+Sample .bashrc setup::
+
+    export RISCV_TOOLCHAIN=<riscv_gcc_install_path>
+    export RISCV_GCC="$RISCV_TOOLCHAIN/bin/riscv64-unknown-elf-gcc"
+    export RISCV_OBJCOPY="$RISCV_TOOLCHAIN/bin/riscv64-unknown-elf-objcopy"
+    export SPIKE_PATH="$RISCV_TOOLCHAIN/bin"
+
+Run bitmanip simulation
```
Comment (taoliug) [inline docs/source/extension_support.rst:32]: can you add a section to explain how to collect functional coverage for B-ext?

Comment (weicaiyang) [reply docs/source/extension_support.rst:34]: Added, thanks

**Comment by weicaiyang** [comment]
> Can you the makefile under doc directory to regenerate the singlehtml doc?

Done, PTAL

**Pull request merged**